NSS-Day-Cohort-49/giffygram-double-rainbows
# Favorite button styling

#### Changes Made
1. Modified file `feed.css` to include inverted favorite button image and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout favorite-button-styling
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Upon clicking favorite button, user will notice that star image file fills and empties container
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
